crates/bevy_ecs/src/change_detection.rs

```diff
@@ -206,6 +217,17 @@ change_detection_impl!(NonSendMut<'a, T>, T,);
 impl_into_inner!(NonSendMut<'a, T>, T,);
 impl_debug!(NonSendMut<'a, T>,);
 
+impl<'a, T: Resource> From<NonSendMut<'a, T>> for Mut<'a, T> {
```

Comment:
This is wrong, should have been:
```suggestion
impl<'a, T: 'static> From<NonSendMut<'a, T>> for Mut<'a, T> {
```
FYI @alice-i-cecile , @maniwani

Reply:
Logged as #5456